LibreNMS/Util/Time.php

```diff
@@ -49,87 +49,38 @@ public static function legacyTimeSpecToSecs($description)
         return isset($conversion[$description]) ? $conversion[$description] : 0;
     }
 
-    public static function formatInterval($interval, $format = 'long')
+    public static function formatInterval($seconds, $format = 'long')
```

Comment:
What was the goal of rewriting formatInterval?  (so I can evaluate the code)

Reply:
Having a more consistent output in Years Days, but constant h:MM:SS format

as before the output would have been like 
1 days 9 hours 59 minutes 59 seconds	; the next second:
1 days 10 hours

With the change:
1 day 9:59:59                                ; the next second:
1 day 10:00:00

Reply:
i'm not sure if this would be such useful.

"1 day 0 hour 0 minute 0 seconds" is kind'a confusing and looks optical corrupt

"1 day" looks better in my eyes

Reply:
ok on this two i'm with you
"1 days 1 hours 1 seconds"
"1 days 1 hours 1 minutes"

they should be
"1 days **0 hours** 1 minutes 30 seconds"
"1 days 1 hours **0 minutes** 1 seconds"

but something like this should stay as it is:
"1 days 1 hours 1 minutes"